e2etest/src/test/java/fk/prof/recorder/utils/AgentRunner.java

```diff
@@ -97,6 +99,12 @@ private void startProcess() throws IOException {
                 .start();
     }
 
+    private String perfCtxArgFrag() {
+        Collection<File> files = FileUtils.listFiles(new File("../perfctx/target"), new RegexFileFilter(PERFCTX_JAR_BASE_NAME_PATTERN), FalseFileFilter.FALSE);
+        if (files.size() != 1) throw new IllegalStateException("Confused about the correct perf-ctx labeling jar, found too many files that look like it: " + files);
```

Comment:
What if no perfctx jar is present? If files.size == 0, the exception message is misleading